src/main/java/uk/gov/digital/ho/hocs/audit/export/infoclient/ExportViewConstants.java

```diff
@@ -2,11 +2,23 @@
 
 public interface ExportViewConstants {
 
+        enum DATE_SECTION {DAY, MONTH, YEAR};
+        int YEAR_DIGITS = 4;
+        int MONTH_DIGITS = 2;
+        int DAY_DIGITS = 2;
+
+        // identifies dates that have the format [0]YYYY-[0]MM-[0]DD
+        String MALFORMED_DATE_REGEX = "^\\d{4,5}\\-0*(?:[1-9]|1[0-2])\\-0*(?:[1-9]|[12][0-9]|3[01])$";
```

Comment:
This allows an unlimited number of zeroes in the month and day column (it accepts `1111-0011-0001`), but only a limited number of zeroes in the year (it rejects `001111-11-11`) -- is that intentional?

Reply:
Good point @zuzak. The initial REGEX was very strict on checking for only one extra preamble zero for year, month and day. With the comments above the regex was changed to its current format which allows for multiple preamble zeros, except for the year. I feel it should probably follow the same pattern, so are you okay with this suggestion?
```suggestion
        String MALFORMED_DATE_REGEX = "^0*\\d{4}\\-0*(?:[1-9]|1[0-2])\\-0*(?:[1-9]|[12][0-9]|3[01])$";
```